test/components/popup.spec.tsx

```diff
@@ -1,17 +1,39 @@
 import React = require('react');
 import ReactDOM = require('react-dom');
-import {selectDom} from 'test-drive';
-import {ClientRenderer, expect, waitFor} from 'test-drive-react';
+import {ClientRenderer, DriverBase, expect, waitFor} from 'test-drive-react';
 import {PopupDemo} from '../../demo/components/popup-demo';
 import {Popup, PopupPositionPoint} from '../../src/components/';
+import {PopupTestDriver} from '../../test-kit/components/popup-driver';
 import {sleep} from '../utils';
 
-const portalId = 'PORTAL';
-const demoContainer = 'POPUP_DEMO_DIV';
+export class PopupDemoTestDriver extends DriverBase {
+    public static ComponentClass = PopupDemo;
+    private popupDriver: PopupTestDriver;
+
+    constructor(public readonly instance: PopupDemo) {
+        super(() => ReactDOM.findDOMNode(instance));
+        this.popupDriver = new PopupTestDriver(instance.getPopup()!);
+    }
+
+    public get container(): HTMLDivElement {
+        return this.select('POPUP_DEMO_DIV');
+    }
+
+    public get portal(): Element {
```

Comment:
It seems to me like portal, content and isPopupPresent functions are redundant. I would do this: 
`public get popup(): PopupTestDriver { return this.popupDriver}` 
and use that in the demo